ydb/docs/en/core/dev/secondary-indexes.md

```diff
@@ -69,6 +69,12 @@ $to_update = (
 UPDATE table1 ON SELECT * FROM $to_update
 ```
 
+{% note info %}
+
+Currently, data updating is possible only using a synchronous secondary index. This is because data modification is possible only in [Serializable](../concepts/transactions.md#modes) transactions, but accessing asynchronous indices would break the guarantees of this transaction mode.
```

Comment:
```suggestion
Currently, data updating is possible only using a synchronous secondary index. This limitation exists because data modification is permitted only in [Serializable](../concepts/transactions.md#modes) transactions, and accessing asynchronous indices would violate the guarantees of this transaction mode.
```